client/src/pages/Landing.js

```diff
@@ -1,8 +1,41 @@
 import React from "react";
+import { makeStyles } from '@material-ui/core/styles';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+import IconButton from '@material-ui/core/IconButton';
+import MenuIcon from '@material-ui/icons/Menu';
```

Comment:
You're not using any of these imports in this file so they don't have to be here.